server/models/schema.sql

```diff
@@ -676,6 +678,8 @@ CREATE TABLE `general_ledger` (
   KEY `period_id` (`period_id`),
   KEY `currency_id` (`currency_id`),
   KEY `user_id` (`user_id`),
+  KEY `cost_center_id` (`cost_center_id`),
+  KEY `principal_center_id` (`principal_center_id`),
```

Comment:
Do we really need a separate key for the principal center id?  I presume the const center ("fee center") table will include a flag as to whether it is principal or not.  So a simple join will give a flag that indicates whether it is a principal center, etc.

EDIT:  I went back and looked at https://github.com/IMA-WorldHealth/bhima/issues/5881.  I know understand why they are both there.